options.go

```diff
@@ -104,6 +104,7 @@ type ClientOptions struct {
 	MaxResumePubInFlight    int // // 0 = no limit; otherwise this is the maximum simultaneous messages sent while resuming
 	Dialer                  *net.Dialer
 	CustomOpenConnectionFn  OpenConnectionFunc
+	AutoAck                 bool
```

Comment:
Sorry to be picky but could we please make this something like 'AutoAckDisabled bool`. The rationale for this is that some users do not use `NewClientOptions` so I'd prefer that the default value for the setting (i.e. `false`) matches the previous behaviour. I realise that this does not read quite as while but feel its worthwhile because based on previous issues some users are just doing `var ClientOptions` (I did add a warning suggesting that this is a bad idea but...). The setter function can remain (just invert the assignment). 
Other than that the PR looks good to go!

Reply:
Sure @MattBrittan . That looks good. Making the change.